src/base/preferences.cpp

```diff
@@ -761,6 +761,26 @@ void Preferences::setWebUICustomHTTPHeaders(const QString &headers)
     setValue("Preferences/WebUI/CustomHTTPHeaders", headers);
 }
 
+bool Preferences::isWebUIReverseProxySupportEnabled() const
+{
+    return value("Preferences/WebUI/ReverseProxySupportEnabled", false).toBool();
+}
+
+void Preferences::setWebUIReverseProxySupportEnabled(const bool enabled)
+{
+    setValue("Preferences/WebUI/ReverseProxySupportEnabled", enabled);
+}
+
+QString Preferences::getWebUITrustedReverseProxiesList() const
+{
+    return value("Preferences/WebUI/TrustedReverseProxiesList", "").toString().trimmed();
```

Comment:
IMO the `trimmed()` should not be here but in webapplication.cpp (where the data is being processed).
The same about `addr.trimmed()` in `setWebUITrustedReverseProxiesList()`.